Dockerfile

```diff
@@ -2,7 +2,7 @@
 FROM registry.access.redhat.com/ubi8/nodejs-16 as builder
 COPY . .
 USER root
-RUN npm config set unsafe-perm true && npm run install:ws && npm run build
```

Comment:
Maybe just verify with @gildub if we should re-introduce this change after this revert goes through?

Reply:
hmm, good call. I'll open an issue

Reply:
https://github.com/konveyor/forklift-ui/issues/872